src/AvaloniaEdit/Utils/ExtensionMethods.cs

```diff
@@ -27,7 +27,7 @@
 
 namespace AvaloniaEdit.Utils
 {
-    internal static class ExtensionMethods
+    public static class ExtensionMethods
```

Comment:
I'm not sure I like the `Utils` namespace to be part of the API. Why do you need this?